ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,60 @@
 periodics:
+- agent: kubernetes
+  labels:
+     ci.openshift.io/role: infra
+  cluster: default
+  cron: "@weekly"
+  decorate: true
+  name: periodic-acme-cert-issuer-for-build01
+  skip_cloning: true
+  spec:
+    containers:
+      - args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            oc --kubeconfig=/etc/build-farm-credentials/kubeconfig whoami
```

Comment:
Are these debugging outputs? Do we need them?

Reply:
It is for debugging ... And I want to keep this if it is OK since we have more and more clusters.
Just want to be clear when I see the the output.